firmware/quintuna/CRIT/src/io/io_shift_register.c

```diff
@@ -0,0 +1,86 @@
+#include "io_shift_register.h"
+#include "hw_gpio.h"
+#include "hw_gpios.h"
+#include "hw_pwmOutput.h"
+#include "hw_pwms.h"
+
+static void runDelay(void)
+{
+    for (uint8_t i = 0; i < 30; i++)
+    {
+    }
+}
```

Comment:
this will get optimized out by the compiler. 
https://stackoverflow.com/a/59927626/6833423